sycl/source/detail/scheduler/commands.hpp

```diff
@@ -329,6 +337,14 @@ class Command {
   bool MFirstInstance = false;
   /// Instance ID tracked for the command.
   uint64_t MInstanceID = 0;
+  /// Code location of command submission to sycl is available.
```

Comment:
What does "to sycl" mean? "to SYCL queue" or "to SYCL API"?

Reply:
done